src/api/spec/models/obs_factory/openqa_job_spec.rb

```diff
@@ -0,0 +1,165 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe ObsFactory::OpenqaJob do
+  describe '.openqa_base_url' do
+    it { expect(ObsFactory::OpenqaJob.openqa_base_url).to eq('http://openqa.opensuse.org') }
+
+    context 'with a configured base_url' do
+      before do
+        stub_const('CONFIG', CONFIG.merge('openqa_base_url' => 'http://configured-url.com'))
+      end
+
+      it { expect(ObsFactory::OpenqaJob.openqa_base_url).to eq('http://configured-url.com') }
+    end
+  end
+
+  describe '.openqa_links_url' do
+    it { expect(ObsFactory::OpenqaJob.openqa_links_url).to eq('https://openqa.opensuse.org') }
+
+    context 'with a configured links_url' do
+      before do
+        stub_const('CONFIG', CONFIG.merge('openqa_links_url' => 'http://links-url.com'))
+      end
+
+      it { expect(ObsFactory::OpenqaJob.openqa_links_url).to eq('http://links-url.com') }
```

Comment:
Looking at [ObsFactory::OpenqaJob](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/obs_factory/openqa_job.rb) it seems like both methods do about the same.
Could we merge openqa_base_url and openqa_links_url? It might even be possible to move the default openqa url to where CONFIG get's instantiated. Then we could even drop these two methods.